lib/backend-api/src/client.rs

```diff
@@ -53,11 +53,18 @@ impl WasmerClient {
     }
 
     pub fn new(graphql_endpoint: Url, user_agent: &str) -> Result<Self, anyhow::Error> {
+        #[cfg(target_family = "wasm")]
```

Comment:
Timeout is available when compiling to `wasix`, which might be useful for certain things, so let's be more specific here and limit to the specific `-unknown`  target.

Reply:
[5dcc9f8](https://github.com/wasmerio/wasmer/pull/4920/commits/5dcc9f8229a53b065ff1a440ccb382fecbc90bb3)